### PR TITLE
es6 migration

### PR DESCRIPTION
**Pull request by reaganthomas**

### CODE REVIEW

**Comment by reaganthomas** [comment]
@prestonvanloon any thoughts?

File: controller/templates/server/api/template.controller.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 (function IIFE() {
   'use strict';
 
-  var errorHandler = require('../../lib/errorHandler');
-
   function exampleMethod(req, res, next) {
-    return errorHandler.notFound(req, res);
+    return res.status(404).json({});
```
Comment (prestonvanloon) [inline controller/templates/server/api/template.controller.js]: I don't like this. Either send back a proper error or only `res.sendStatus(404);` 

http://jsonapi.org/format/#fetching-resources-responses-404

File: templates/package.json [hunk 2]
```diff
@@ -36,11 +31,7 @@
     "supertest": "^1.1.0"
   },
   "scripts": {
-    "start": "gulp clean && gulp lint && gulp move && node server",
+    "start": "gulp lint && node server",
```
Comment (prestonvanloon) [inline templates/package.json]: No need to `gulp lint` in production

**Comment by prestonvanloon** [comment]
Fix https://github.com/reaganthomas/slush-athena/blob/feature/update_to_es6/templates/server/routes.js#L7

**Comment by prestonvanloon** [comment]
ok :+1:

**Pull request merged**